src/battle/area/pra/actor/albino_dino.c

```diff
@@ -124,83 +124,83 @@ EvtScript N(handleEvent) = {
         EVT_CASE_OR_EQ(EVENT_HIT_COMBO)
         EVT_CASE_OR_EQ(EVENT_HIT)
             EVT_SET_CONST(LVar0, 1)
-            EVT_SET_CONST(LVar1, ANIM_AlbinoDino_Anim08)
+            EVT_SET_CONST(LVar1, ANIM_AlbinoDino_Pain)
             EVT_EXEC_WAIT(EVS_DoNormalHit)
         EVT_END_CASE_GROUP
         EVT_CASE_EQ(EVENT_BURN_HIT)
             EVT_SET_CONST(LVar0, 1)
-            EVT_SET_CONST(LVar1, ANIM_AlbinoDino_Anim0A)
-            EVT_SET_CONST(LVar2, ANIM_AlbinoDino_Anim0B)
+            EVT_SET_CONST(LVar1, ANIM_AlbinoDino_BurntPain)
+            EVT_SET_CONST(LVar2, ANIM_AlbinoDino_BurntDead)
             EVT_EXEC_WAIT(DoBurnHit)
         EVT_CASE_EQ(EVENT_BURN_DEATH)
             EVT_SET_CONST(LVar0, 1)
-            EVT_SET_CONST(LVar1, ANIM_AlbinoDino_Anim0A)
-            EVT_SET_CONST(LVar2, ANIM_AlbinoDino_Anim0B)
+            EVT_SET_CONST(LVar1, ANIM_AlbinoDino_BurntPain)
+            EVT_SET_CONST(LVar2, ANIM_AlbinoDino_BurntDead)
             EVT_EXEC_WAIT(DoBurnHit)
             EVT_SET_CONST(LVar0, 1)
-            EVT_SET_CONST(LVar1, ANIM_AlbinoDino_Anim0B)
+            EVT_SET_CONST(LVar1, ANIM_AlbinoDino_BurntDead)
             EVT_EXEC_WAIT(EVS_DoDeath)
             EVT_RETURN
         EVT_CASE_EQ(EVENT_SPIN_SMASH_HIT)
             EVT_SET_CONST(LVar0, 1)
-            EVT_SET_CONST(LVar1, ANIM_AlbinoDino_Anim08)
+            EVT_SET_CONST(LVar1, ANIM_AlbinoDino_Pain)
             EVT_EXEC_WAIT(DoSpinSmashHit)
         EVT_CASE_EQ(EVENT_SPIN_SMASH_DEATH)
             EVT_SET_CONST(LVar0, 1)
-            EVT_SET_CONST(LVar1, ANIM_AlbinoDino_Anim08)
+            EVT_SET_CONST(LVar1, ANIM_AlbinoDino_Dead)
```

Comment:
Some changes around here are wrong. 8 is pain, but you changed 8 to dead here. also the build doesn't match (jenkins failed, and your local build should be failing too) - please fix these